.github/workflows/release.yaml

```diff
@@ -94,6 +94,12 @@ jobs:
                 --build-arg TAG=$TAG \
                 ${dir}
           done
+
+  scan:
+    name: Scan images
+    runs-on: ubuntu-24.04
+    needs: build
+    steps:
```

Comment:
By splitting a job, we can't refer `BUILD` file generated at `build` job  at `scan` job.
You can use `GITHUB_OUTPUT` variable to pass key-value pairs across jobs.